docs/customising/make_a_new_theme.md

```diff
@@ -96,6 +105,32 @@ right place within your Alaveteli installation.
 When it's finished, you can see that your theme (that is, all the files from
 the repo) has been reproduced in `lib/themes/`.
 
+<div class="attention-box helpful-hint">
+  <p>
+    If you're just following this guide to see what is possible and have no
+    interest in saving the changes you're about to make, you can skip this bit!
+  </p>
+  <p>
+    On the command line (back on your own machine rather than inside Vagrant
+    box), `cd` to your theme's folder inside `lib/themes/`. This is where your
```

Comment:
These backticks don't render as code for me; I thought you had to stick to HTML until you close the first opening tag? As before, might be my env. Worth checking though.

![screen shot 2016-04-01 at 15 04 55](https://cloud.githubusercontent.com/assets/282788/14209256/1cc3d98c-f81b-11e5-8fc8-e980d7ef5435.png)